ParseSwift.playground/Pages/2- Finding Objects.xcplaygroundpage/Contents.swift

```diff
@@ -10,7 +10,7 @@ PlaygroundPage.current.needsIndefiniteExecution = true
 
 initializeParse()
 
-struct GameScore: ParseSwift.ObjectType {
+struct GameScore: ParseSwift.ParseObject {
```

Comment:
I don't have a problem with `ParseSwift.Object`, but doing so would require making `Object` a nested typealias of `ParseSwift`, and that would make both `Object` and `ParseSwift.Object` valid.

I think it would be better to just have `ParseObject`, but I'm not against the other approach.

Reply:
I’m on my iPad, so I can’t confirm this, but it sounds like how currently have it setup, `ParseObject` is valid now? If so, then I take back my comments as we can just encourage developers to use `ParseObject`and `ParseUser`. I think from our previous discussion we were trying to stay away from `Object` and `User`  as they are too generic and can easily conflict with another framework when made public

Reply:
Nvm, I see you demonstrated it in your fixes. I think we are good with ParseObject and ParseUser